specs/phase0/beacon-chain.md

```diff
@@ -1556,7 +1580,10 @@ def process_registry_updates(state: BeaconState) -> None:
         if is_eligible_for_activation_queue(validator):
             validator.activation_eligibility_epoch = get_current_epoch(state) + 1
 
-        if is_active_validator(validator, get_current_epoch(state)) and validator.effective_balance <= EJECTION_BALANCE:
+        if (
+            is_active_validator(validator, get_current_epoch(state))
+            and validator.effective_balance <= EJECTION_BALANCE
+        ):
```

Comment:
this was just a formatting change?

im assuming the linter was happy either way

Reply:
It changes to `config.EJECTION_BALANCE` by the pyspec builder, and gets too long for the line.